<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>672</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>230</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Зачисление абитуриентов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/images/appIcon.png</normaloff>
    <normalon>:/images/appIcon.png</normalon>:/images/appIcon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="personalDataFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>621</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <widget class="QLabel" name="personalDataLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Анкетные данные:</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>521</width>
       <height>161</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>10</number>
     </property>
     <widget class="QWidget" name="">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ФИО:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLine">
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>45</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="birthdateLabel">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Дата рождения:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="birthdateEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QFormLayout" name="formLayout_4">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>200</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="sexLabel">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пол:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="sexHL">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QRadioButton" name="maleRB">
           <property name="text">
            <string>муж</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="femaleRB">
           <property name="text">
            <string>жен</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="otherRB">
           <property name="text">
            <string>другое</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="schoolFrame">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>451</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>431</width>
       <height>275</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="medalCB">
        <property name="font">
         <font>
          <family>Tahoma</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Имеется золотая/серебрянная медаль</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="averageScoreLabel">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Средняя оценка аттестата:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="averageScoreDB"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="foreignLangLabel">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Изучаемый иностранный язык:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="engCB">
              <property name="text">
               <string>Английский</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="gerCB">
              <property name="text">
               <string>Немецкий</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="frCB">
              <property name="text">
               <string>Французский</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="korCB">
              <property name="text">
               <string>Корейский</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chinCB">
              <property name="text">
               <string>Китайский</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="japCB">
              <property name="text">
               <string>Японский</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1224</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
